### ISSUE TITLE
Masks not rendered properly

### ISSUE THREAD

**Issue by thibnes** [vvvv/SVG#482]
Using this file as example 
[testSvgMasks.zip](https://github.com/vvvv/SVG/files/3261803/testSvgMasks.zip)

the generated jpeg looks like this 
![output](https://user-images.githubusercontent.com/11884292/59037667-0602b200-8872-11e9-9dd2-a6cf548fdb86.jpg)

Is there a special setup to handle masks?

**Comment by mrbean-bremen** [comment]
I will have a look later, but this is most certainly not supported or a bug.

**Comment by mrbean-bremen** [comment]
Yes, masks are unsupported - the related W3C examples also fail. There is an empty class `SvgMask`, so I guess an implementation has been considered some time...
So I think you are out of luck here, as I doubt that mask implementation is something trivial enouogh to be added soon.

**Comment by thibnes** [comment]
> Yes, masks are unsupported - the related W3C examples also fail. There is an empty class `SvgMask`, so I guess an implementation has been considered some time...
> So I think you are out of luck here, as I doubt that mask implementation is something trivial enough to be added soon.

Alright, thank you for your quick response.

**Comment by Zakkon** [comment]
I found a simple workaround, for anyone interested. To get masks to atleast be written into the .svg files, create a class like this:

`[SvgElement("mask")]
public class SvgMask2 : SvgMask { }`
And use that instead of `SvgMask`. Remember to give it an ID so you can reference it later.
 
And to make a `SvgElement` use the said mask, simply add a custom attribute to it, using the ID you gave the mask:
`element.CustomAttributes.Add("mask", "url(#maskID)");`

This will obviously not render when converting to bitmap, but it won't throw an error either. Useful if all you care about is the .svg file.

**Comment by mrbean-bremen** [comment]
Yes, I also added the attribute to the (empty) SvgMask class, and this fixes at least the writing (though not the functionality, of course). I thought it not worth a separate commit without the mask working, but that was obviously wrong. I will add this tomorrow.

**Comment by mrbean-bremen** [comment]
Thanks for the hint, I committed the change now!
